src/components/common/Sidebar/SidebarLinks.utils.js

```diff
@@ -42,11 +42,20 @@ export const bottomSharedLinks = [
   },
 ];
 
+/**
+ * Khaleel Musleh
+ * Uncommented all the links which are non-functional, they are all available in the devLinks below
+ */
+
 export const menteeLinks = [
-  {
-    key: '/assignments',
-    label: 'My Assignments',
-  },
+  // {
```

Comment:
let's go ahead and remove this before merging.

Reply:
Removed it on last commit